src/pe_reports/report_generator.py

```diff
@@ -295,12 +324,7 @@ def generate_reports(db, datestring, data_directory, output_directory):
     """Process steps for generating report data."""
     agencies = []
     contents = os.walk(data_directory)
-
-    # ISSUE - Pathing Error???
-    f = open(
-        "/Users/schmelzs/Repos/PE_Report_Scripts/pe-reports_v1.0/pe-reports/src/pe_reports/org_names.json"
-    )
-    names_obj = json.load(f)
+    names_obj = load_customers()
```

Comment:
If this is an empty dictionary you should bail out since that means organization data couldn't be loaded.

Reply:
@mcdonnnj , 

I made an adjustment per commit [cb45751](https://github.com/cisagov/pe-reports/pull/33/commits/cb45751df9557cabc8534c2ca9bc79861ab97cf9) that addresses the need to verify the org_names.json file is present and has data in the file.

Reply:
There is still no check that there is a problem with the returned value from `load_customers()`. If there is a problem accessing `org_names.json` you currently output a log message, return a `None`, and then in https://github.com/cisagov/pe-reports/blob/8f982085d507fa59b328347175899225f57153ea/src/pe_reports/report_generator.py#L333
you immediately try and access the result without verifying it is a non-empty dictionary.